packages/paste-core/components/form/__tests__/formTextarea.test.tsx

```diff
@@ -0,0 +1,240 @@
+import * as React from 'react';
+import {render} from 'react-dom';
+import renderer from 'react-test-renderer';
+import {shallow, ReactWrapper, mount} from 'enzyme';
+import {axe} from 'jest-axe';
+import {Theme} from '@twilio-paste/theme';
+import {FormHelperText, FormLabel, FormTextArea} from '../src';
+
+const NOOP = (): void => {};
+
+describe('FormTextArea render', () => {
+  it('it should render', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <FormTextArea id="textarea" onChange={NOOP} />
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('it should render with readOnly', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <FormTextArea id="textarea" onChange={NOOP} readOnly />
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
prefer enzyme to validate these props over snapshots

Reply:
Potentially if you're doing the enzyme test down at https://github.com/twilio-labs/paste/pull/345/files#diff-60403f775a91aa64baef1b2aed75204fR124 you can probably just remove this snapshot test entirely

Reply:
Removed and just left the initial render snapshot.